client_delivery/shared/src/commonMain/kotlin/presentation/login/composable/CustomBottomSheet.kt

```diff
@@ -0,0 +1,119 @@
+package presentation.login.composable
+
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.animation.core.tween
+import androidx.compose.animation.fadeIn
+import androidx.compose.animation.fadeOut
+import androidx.compose.animation.slideInVertically
+import androidx.compose.animation.slideOutVertically
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.Stable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.beepbeep.designSystem.ui.composable.modifier.noRippleEffect
+import com.beepbeep.designSystem.ui.theme.Theme
+
+
+@Composable
+fun CustomBottomSheet(
```

Comment:
i think u can find another name , something like `BpBottomSheet` and try to make it usable

Reply:
as we agreed, we will refactor the name to be some thing of scafold